packaging/systemd/cloudstack-usage.default

```diff
@@ -15,8 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-JAVA_OPTS="-Dpid=$$ -Xms256m -Xmx2048m"
```

Comment:
@GabrielBrascher What's the issue, this should work?

Reply:
Nevermind, re-read the description. Alright keep this change, I'll advise you a simpler fix.

Reply:
Btw, this without quotes would work